language/tokens.ts

```diff
@@ -147,6 +147,17 @@ const commonMatchers: Matcher[] = [
       type: `word`
     }
   },
+  {
+    name: `WHEN-GROUP`,
+    match: [
+      { type: `word`, match: (word) => word.toUpperCase() === `WHEN` },
+      { type: `minus` },
+      { type: `word` },
```

Comment:
```suggestion
      { type: `word`, match: (word) => [`IN`, `IS`].includes(word.toUpperCase())},
```

This is more correct - although don't think it is used anywhere?